docs/content/documentation/deployment/zeabur.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "Zeabur"
+weight = 50
++++
+
+In this tutorial, we'll guide you through the process of deploying your Zola site onto the [Zeabur](https://zeabur.com). Zeabur provides a seamless deployment experience, with automatic SSL certificates and global-edge network delivery. Let's get started!
+
+## Prerequisites
+
+- A Zola site project on your local machine.
+- A GitHub account and your Zola site project repository hosted on GitHub.
+- A Zeabur account. If you don't have one, sign up at [here](https://zeabur.com/login).
+
+## Step 1: Create a Project on Zeabur
+
+1. Log in to your Zeabur account.
+2. Navigate to the dashboard and click on the **Create Project** button.
+3. Follow the on-screen instructions to set up a new project.
+
+## Step 2: Push Your Zola Files to GitHub
+
+1. Initialize a Git repository in your Zola project folder if you haven't already:
+
+    ```bash
+    git init
+    git add .
+    git commit -m "Initial commit"
+    ```
+
+2. Push your Zola project to GitHub:
+
+    ```bash
+    git remote add origin <your-github-repo-url>
+    git branch -M main
+    git push -u origin main
+    ```
+
+Replace `<your-github-repo-url>` with the URL of your GitHub repository.
+
+## Step 3: Create a Service on Zeabur
+
+1. Back in your Zeabur dashboard, click on **Create Service**.
+2. Choose the **git** option to connect your GitHub repository.
+
+## Step 4: Select Your Zola Repository
+
+1. From the list of repositories, select the repository where your Zola project is located.
+
+## Step 5: Automatic Deployment
+
+Zeabur will automatically detect that you're deploying a Zola project and will handle the deployment process for you without any additional configuration needed.
```

Comment:
how does it select a version?

Reply:
Thank you for the heads-up. We have introduced the `ZOLA_VERSION` environment variable, allowing users who wish to deploy with a specific version of Zola on Zeabur to easily select the version they require.

https://github.com/getzola/zola/pull/2408/commits/ac06d42c25e6c375c0fb623cc03de08f72495a3a